scripts/linters/pylint_extensions_test.py

```diff
@@ -3257,3 +3257,67 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class DisallowBlankLinesBelowFunctionDefinitionCheckerTests(unittest.TestCase):
```

Comment:
Can you also add a test for the case where function definition might extend over 2 or more lines? See an example of that below (This is a pretty common case of function def in our codebase, and the check should work correctly for it):
```python
def this_is_a_test_name_that_spans_across_multiple_lines(
        attribute_name_too_long_to_fit_in_single_line):
    pass
```

Reply:
Thanks a lot @sajalasati for this test case, I totally missed it. I have fixed it now, I will update my PR by tommorow as more lint errors have appeared now.

Reply:
The while loop which you had earlier could have helped you for these kinds of cases. But while using a loop, you should ensure that it does not end up in some infinite looping condition - this was my [comment](https://github.com/oppia/oppia/pull/11980#discussion_r593126967) in the earlier review. Kindly think properly about your approach once and run it on some actual (and variety of) files locally to verify.

Reply:
Yes @sajalasati I think i have corrected it and I will update my PR tommorow. I have used for loop this time to iterate  through each and every line of the function untill a line ending with colon is found, after which I just use check for the length of next line. This code is working fine also, I have ran it over the whole oppia folder using 'python -m scripts.linters.pre_commit_linter --path /home/mayank/opensource/oppia' and it has ran also. Many lint errors have also been found and I will update my PR tommorow after correcting them.